### DISCUSSION TITLE
manual installation on Windows

### DISCUSSION THREAD

**Discussion by o-dinn** [vcsjones/AzureSignTool#212] [Q&A]
ENV: Server 2019, .NET Core SDK 8.0, NuGet 2.8.5.208, PowerShell 5.1.17763.4974, PowerShellGet 1.0.0.1

I have tried to get the AzureSignTool package installed using the --global but default install does not have a NuGet source defined;

`dotnet : Unhandled exception: Microsoft.DotNet.Cli.NuGetPackageDownloader.NuGetPackageInstallerException: No NuGet sources are defined or enabled`

I downloaded the nuget package locally and was able to get the PowerShell command to return no errors

`Invoke-WebRequest -Uri https://globalcdn.nuget.org/packages/azuresigntool.4.0.1.nupkg -OutFile C:\temp

Install-Package AzureSignTool -Source C:\temp`

However AzureSignTool is still not recognized as a command.  Is there a step I'm missing in getting the package installed and recognized as a cmdlet?

`AzureSignTool : The term 'AzureSignTool' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct 
and try again.`

**Comment by shermanericts** [comment]
I"m also having an issue installing this using nuget.
I try this, and get the following:

```
C:\Program Files\azuresigntool>dotnet tool install -g AzureSignTool
azuresigntool is not found in NuGet feeds C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\".
```
